templates/validate-h-card.html.php

```diff
@@ -74,36 +113,55 @@
 				</div>
 				<?php endif ?>
 
-				<p class="property-block-name">Email</p>
 				<?php if (Mf2\hasProp($hCard, 'email')): ?>
+				<p class="property-block-name">Email</p>
 				<ul>
 					<?php foreach ($hCard['properties']['email'] as $email): ?>
 					<li><a href="<?= $email ?>"><?= $email ?></a></li>
 					<?php endforeach ?>
 				</ul>
-				<?php else: ?>
-				<div class="empty-property-block">
-					<p>Add your Email! <code class="pull-right">&lt;a rel=&quot;me&quot; class=&quot;u-email&quot;>…&lt;/a></code></p>
-				</div>
 				<?php endif ?>
 
-				<p class="property-block-name">Note</p>
 				<?php if (Mf2\hasProp($hCard, 'note')): ?>
+				<p class="property-block-name">Note</p>
 				<p><?= Mf2\getProp($hCard, 'note') ?></p>
-				<?php else: ?>
-				<div class="empty-property-block">
-					<p>Add a note/bio! <code class="pull-right">&lt;p class=&quot;p-note&quot;>…&lt/p></code></p>
-				</div>
 				<?php endif ?>
 
+				<?php
+				foreach ($optional_properties as $name => $label) {
+					if (Mf2\hasProp($hCard, $name)) {
+						echo sprintf('<p class="property-block-name">%s</p> <ul>', $label);
+						foreach (Mf2\getPlaintextArray($hCard, $name) as $value) {
+							echo sprintf('<li>%s</li>', $value);
+						}
+						echo '</ul>';
+					}
+				}
+				?>
+
+				<h2> Additional Properties </h2>
+				<p> If you are publishing additional information about yourself, these are some h-card properties you can add: </p>
```

Comment:
I feel like this phrasing is weird if neither email nor note will be described below (because of them already being available). Is that just me?

Reply:
Yeah, it felt a bit odd. I think we can remove that heading and include email and note in the optional properties, then link to the wiki at the bottom for reference.